axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/TimesheetProjectServiceImpl.java

```diff
@@ -144,4 +147,21 @@ public TimesheetProjectServiceImpl(
 
     return invoiceLineList;
   }
+
+  @Override
+  public TimesheetLine generateTimesheetLine(
+      Timesheet timesheet, ProjectPlanningTime projectPlanningTime) throws AxelorException {
+    TimesheetLine timesheetLine = super.generateTimesheetLine(timesheet, projectPlanningTime);
+    if (timesheetLine != null) {
+      if (timesheetLine.getTeamTask().getTimeInvoicing()
```

Comment:
The 'teamTask' field is not a required field, why there is no null check on it ?